common/conf.example

```diff
@@ -6,12 +6,19 @@ adminMail=admin@example.com
 basePath=/srv
 
 # Site name = user name that executes the site.
-userNames="org1 org2"
+userNames=""
+
+# Use letsencrypt for HTTPS? If so, DNS records need to be accurate before installing!
```

Comment:
Accurate in what way?

Perhaps we can write that DNS records must point to the server running the script in order for installation to work?